build.gradle.kts

```diff
@@ -32,6 +38,9 @@ intellij {
 
     // Plugin Dependencies. Uses `platformPlugins` property from the gradle.properties file.
     plugins.set(properties("platformPlugins").split(',').map(String::trim).filter(String::isNotEmpty))
+
+    plugins.set(listOf("com.intellij.java"))
```

Comment:
I _think_ this may be a leftover from before I merged with some other changes. I remember not being able to get PSI to work without setting that plugin. Removing it works fine, I'm guessing due to the thing in `gradle.properties`